data/org.gnome.Software.mime.xml

```diff
@@ -3,7 +3,7 @@
   <mime-type type="x-content/ostree-software">
     <comment>Endless Apps Repository</comment>
     <comment xml:lang="es">Repositorio de Aplicaciones para Endless</comment>
-    <comment xml:lang="id_ID">Gudang Aplikasi Endless</comment>
+    <comment xml:lang="id_ID">Repository Aplikasi Endless</comment>
```

Comment:
Talking with Nieke, Repository is a more approachable word and it makes more sense in the context, since Gudang means warehouse in english.